community-operators/service-binding-operator/service-binding-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: service-binding-operator.v0.1.1-352
+- currentCSV: service-binding-operator.v0.1.1-349
```

Comment:
Hi @J0zi . Had a doubt. If we give a lesser version number , in this case 349 , would that be reflected on OperatorHub or we can only go version ++

Reply:
@Avni-Sharma, it will work. But it is not recommended. If look to the future changes we are preparing, it can make chaos in this setup. Please consider higher version.